tables/de-g2.ctb

```diff
@@ -8,7 +8,7 @@
 #+grade:2
 #+direction:forward
 #
-#-has-nocross: yes
+#-has-nocross always: yes
```

Comment:
Is this a new meta data attribute? Is this documented anywhere?

Reply:
Oops, query-replace error.

Reply:
Fixed